modules/public/views.py

```diff
@@ -315,7 +432,9 @@ def external(attrs, new=False):
         'is_linux': (
             'Linux' in flask.request.headers.get('User-Agent', '') and
             'Android' not in flask.request.headers.get('User-Agent', '')
-        )
+        ),
+
+        'errors': errors
     }
 
     return flask.render_template(
```

Comment:
If we do have an API communication error, could we please return a non-200 status? Since all the information for that page comes from the API, I don't think it counts has a successful response if the information isn't there. We can keep the page running through the same template - i.e. looking relatively pretty - but I think its status should probably be one of:

```
502 Bad Gateway
503 Service Unavailable
504 Gateway Timeout
```

Reply:
done every template reterns a status code